modules/cats-effect/src/test/scala/pureconfig/module/catseffect/CatsEffectSuite.scala

```diff
@@ -11,39 +11,41 @@ import java.nio.file.{ Path, Paths }
 
 import com.typesafe.config.ConfigFactory
 
-class CatsEffectSuite extends BaseSuite {
+final class CatsEffectSuite extends BaseSuite {
 
-  case class SomeCaseClass(somefield: Int, anotherfield: String)
+  private case class SomeCaseClass(somefield: Int, anotherfield: String)
+
+  private implicit val ioCS = IO.contextShift(scala.concurrent.ExecutionContext.global)
 
   private def getPath(classPathPath: String): Path = {
     val resource = getClass.getClassLoader.getResource(classPathPath)
     Paths.get(resource.toURI)
   }
 
   "loadConfigF" should "run successfully when correctly formatted file is in place" in {
-    val load = loadConfigF[IO, SomeCaseClass]
+    val load = Blocker[IO].use(loadConfigF[IO, SomeCaseClass])
```

Comment:
We should probably use a single `Blocker` for all of the suite. Something like this would be enough for now:
```scala
val blocker: Blocker =
  Blocker.liftExecutionContext(scala.concurrent.ExecutionContext.global)
```